test/basekernels/fbm.jl

```diff
@@ -13,7 +13,14 @@
 
     test_interface(k)
     @test repr(k) == "Fractional Brownian Motion Kernel (h = $(h))"
-    test_ADs(FBMKernel; ADs=[:ReverseDiff, :Zygote])
-    @test_broken "Tests failing for kernelmatrix(k, x) for ForwardDiff"
+    test_ADs(FBMKernel; ADs=[:ReverseDiff])
+
+    # Tests failing for ForwardDiff and Zygote.
+    # Related to: https://github.com/FluxML/Zygote.jl/issues/1036
+    @test_broken !isinf(ForwardDiff.gradient(x -> x[1]^x[2], [0.0, 0.9])[1])
+    @test_broken !isinf(
```

Comment:
Looks like you might be missing a line:
```suggestion
    f(x, y) = x ^ y
    @test_broken !isinf(
```

Reply:
Indeed I am -- should have been more careful